bowgenerator_test.go

```diff
@@ -22,7 +22,7 @@ func TestGenerator(t *testing.T) {
 	})
 
 	t.Run("with missing data", func(t *testing.T) {
-		b, err := NewGenBow(1000000, GenSeriesOptions{MissingData: true})
+		b, err := NewGenBow(100, GenSeriesOptions{MissingData: true})
```

Comment:
Why decreasing this value ?

Reply:
The previous value was arbitrary very high for no reason at all. Decreasing it will speed up and reduce memory consumption of this test